code/game/machinery/doors/airlock.dm

```diff
@@ -751,27 +751,23 @@
 		switch(security_level)
 			if(AIRLOCK_SECURITY_NONE)
 				if(istype(C, /obj/item/stack/sheet/metal))
-					var/obj/item/stack/sheet/metal/S = C
-					if(S.get_amount() < 2)
-						to_chat(user, "<span class='warning'>You need at least 2 metal sheets to reinforce [src].</span>")
+					if(!C.tool_start_check(user, amount=2))
 						return
 					to_chat(user, "<span class='notice'>You start reinforcing [src].</span>")
-					if(do_after(user, 20, 1, target = src))
-						if(!panel_open || !S.use(2))
+					if(C.use_tool(src, user, 20, volume=50, amount=2))
```

Comment:
Doesn't volume come after amount?

Reply:
It doesn't really matter for keyword args, and I prefer to make amount the last one, so it would be noticeable.

Reply:
@tgstation/commit-access Opinions on this? I feel it sets a bad precedent to mix the ordering of proc arguments, key-worded or not.

Reply:
Ordering on proc arguments is already a huge mess. I mean, have you seen how inconsistently `do_after` is used? I have, and it wasn't pretty.

Reply:
why are you even doing key-worded args here anyway

Reply:
because not every `use_tool` call uses those

Reply:
Broken Windows. Fix the windows. Stop repeating bad patterns.

https://en.wikipedia.org/wiki/Broken_windows_theory

Reply:
this is trivially fixable by search and replace so I'm not sure why people are bikeshedding over it

Reply:
Because someone invited the entire coderbus here.

Reply:
@ACCount12 There always comes a point where you should ask the question "Is this the hill I want to die on?" and right now the amount of time it would have taken you to fix what was asked is much  less time than the 2 days that this discussion has been going on for.

If this is the sort of thing that you are going to push back against with this much effort then I have to ask how much more are you going to push? How often are you going to push?

Reply:
When I made this PR, I haven't expected to get an entire coderbus to be dropped on my ass. Fuck off.

Reply:
@ACCount12 We are happy to have your work, but I suggest taking a break for a while to cool off for your own sake. Let's revisit this later.